ports/libsimpleservo/api/src/lib.rs

```diff
@@ -129,6 +129,12 @@ pub trait HostTrait {
     fn get_clipboard_contents(&self) -> Option<String>;
     /// Sets system clipboard contents
     fn set_clipboard_contents(&self, contents: String);
+    /// FIXME
+    fn to_immersive_mode(&self);
+    /// FIXME
+    fn flush_xr(&self);
```

Comment:
this is stuff leftover from @paulrouget's code, I'm not sure if we should remove these. I think I'm going to leave the immersive cube thing intact until we have a proper immersive mode.

Reply:
Are these functions used at any point?
When do swapBuffers and makeCurrent end up being called for XR?

to_immersive_mode is probably not necessary, in the UWP app we will notice that the app is navigated away.

Reply:
>  When do swapBuffers and makeCurrent end up being called for XR?

Probably in end_frame in OpenXR.

Reply:
I'll get rid of these changes. With that I don't think there's any of your original code left. I wasn't sure how much of it was WIP for other stuff.

Reply:
That was just for some short term testing.

Reply:
Done